cmd/kogito/command/install/data_index.go

```diff
@@ -92,6 +97,16 @@ func (i *installDataIndexCommand) Exec(cmd *cobra.Command, args []string) error
 		return err
 	}
 
+	persistenceProvider := v1alpha1.InfinispanPersistenceProvider
+	if i.flags.kogitoDataIndexPersistence == "mongodb" {
```

Comment:
Can you move this value to a constant?

```suggestion
	if i.flags.kogitoDataIndexPersistence == string(v1alpha1.MongoDBPersistenceProvider) {
```